src/parachain/staked-relayer.ts

```diff
@@ -113,45 +89,25 @@ export interface StakedRelayerAPI extends TransactionAPI {
      * @returns The maximum SLA score, a positive integer
      */
     getMaxSLA(): Promise<number>;
-    /**
-     * @returns The number of blocks to wait until eligible to vote
-     */
-    getStakedRelayersMaturityPeriod(): Promise<BlockNumber>;
     /**
      * @param planckStake Stake amount (denoted in Planck) to register with
      */
     register(planckStake: BN): Promise<void>;
-    /**
-     * @param depositPlanck Deposit required to suggest the status update
-     * @param statusCode Suggested BTC Parachain status
-     * @param message Message detailing reason for status update
-     * @param addError If the suggested status is Error, this set of ErrorCode indicates which error is to be added to the Errors mapping.
-     * @param removeError ErrorCode to be removed from the Errors list.
-     * @param blockHash When reporting an error related to BTC-Relay, this field indicates the affected Bitcoin block (header).
-     */
-     suggestStatusUpdate(
-        depositPlanck: BN,
-        statusCode: StatusCode,
-        message: string,
-        addError?: ErrorCode,
-        removeError?: string,
-        blockHash?: string,
-    ): Promise<void>;
-    /**
-     * @param depositPlanck Deposit required to suggest the status update
-     * @param blockHash When reporting an error related to BTC-Relay, this field indicates the affected Bitcoin block (header).
-     * @param message Message detailing reason for status update
-    */
-    suggestInvalidBlock(deposit: BN, blockHash: string, message: string): Promise<void>
-    /**
-     * @param statusUpdateId Identifier of the `StatusUpdate` voted upon in `ActiveStatusUpdates`.
-     * @param approve `true` or `false`, depending on whether the Staked Relayer agrees or disagrees with the suggested `StatusUpdate`.
-     */
-    voteOnStatusUpdate(statusUpdateId: BN, approve: boolean): Promise<void>;
     /**
      * Deregister the Staked Relayer
      */
     deregister(): Promise<void>;
+    /**
+     * A Staked Relayer reports misbehavior by a Vault, providing a fraud proof
+    * (malicious Bitcoin transaction and the corresponding transaction inclusion proof).
+     * @remarks If `txId` is not set, the `merkleProof` and `rawTx` must both be set.
+    * 
+    * @param vault_id The account of the vault to check.
+    * @param tx_id The hash of the transaction
+    * @param merkle_proof The proof of tx inclusion.
+    * @param raw_tx The raw Bitcoin transaction.
+     */
+    reportVaultTheft(vaultId: string, btcTxId?: string, merkleProof?: Bytes, rawTx?: Bytes): Promise<void>;
```

Comment:
In this case, we need the btcTxId since I don't think there is any way for us to figure that out from just the information we have available in the lib at that point.